bepasty_cli/cli.py

```diff
@@ -38,6 +50,15 @@
     'fname',
     help='Filename for piped input.')
 @click.option(
+    '-m',
+    '--max-lifetime',
+    type=(int, click.Choice(LIFETIME_CHOICES)),
+    default=(1, 'f'),
+    help='Lifetime for the file that is uploaded. Example: "-m 2 d" (two days). If this '
+         'option is not set, the uploads lifetime is "forever". Multiplier and unit are '
+         'space separated. Unit has to be one of these: "min" (minutes), "h" (hours), '
```

Comment:
Yes, that's right. But this way we can use an existing declarative programming deature of clicks.
If we'd go with `-m 10d` we'd have to parse and error-message ourselfs. I think the usual (w/o space) style stems from the complexity of traditional cmdline parsing.

If you think it's important, I would try to code a [custom clicks type](http://click.pocoo.org/5/parameters/#implementing-custom-types).